modules/tinymce/src/core/main/ts/textpatterns/core/PatternTypes.ts

```diff
@@ -53,7 +53,7 @@ export type Pattern = InlinePattern | BlockPattern;
 
 export interface DynamicPatternContext {
   readonly text: string; // the string from the start of the block to the cursor
-  readonly block: Node; // the parent block node
+  readonly block: Element; // the parent block node
```

Comment:
Maybe change the comment as well to `the parent block element`

Reply:
Good point. 

- [ ] change comment.